packages/apidom-ns-workflows-1/src/refractor/visitors/workflows-1/sourceDescription/index.ts

```diff
@@ -0,0 +1,18 @@
+import stampit from 'stampit';
```

Comment:
Please change the name of the directory from `workflows-1/sourceDescription` to `workflows-1/source-description`. By convention all directories are `dash-cased`.